Cargo.toml

```diff
@@ -49,4 +50,4 @@ harness = false
 bench-complex-all = []
 circom-2 = []
 ethereum = ["ethers-core"]
-default = ["ethereum"]
+default = ["circom-2", "ethereum"]
```

Comment:
I'm not sure what to keep as default. @gakonst please suggest!

Reply:
Yep this is good. I think we should probably remove the Circom 1 code if nobody uses it anymore